src/ripple/nodestore/backend/NuDBFactory.cpp

```diff
@@ -38,7 +38,9 @@ namespace NodeStore {
 class NuDBBackend : public Backend
 {
 public:
-    static constexpr std::size_t currentType = 1;
+    static constexpr std::uint64_t currentType = 1;
+    static constexpr std::uint64_t deterministicType = 0x5348524400000000ll;
+    static constexpr std::uint64_t deterministicMask = 0xFFFFFFFF00000000ll;
```

Comment:
Nitpick:  A `ull` suffix would be more consistent.

Reply:
Done